stats.tex

```diff
@@ -0,0 +1,50 @@
+%% Stats
+The \verb|sympy.stats| module provides random variable types and methods for
+determination of statistical properties of expressions involving random
+variables. The variables can be either continuous or discontinuous, which are
+further divided into finite and infinite. They are associated
+with probability densities on corresponding domains and internally defined
+in terms of probability spaces. 
+Apart from the possibility of defining the random variables from user supplied
+density distribution, SymPy provides definitions of most common
+distributions, including \texttt{Uniform}, \texttt{Poisson}, \texttt{Normal},
+\texttt{Binomial}, \texttt{Bernoulli}, and many others.
+
+Properties of random expressions can be calculated using, e.g.,
+\texttt{E} and \texttt{variance} to calculate expectation and variance.
+Conditions on random variables can be defined with inequalities, equalities,
+and logical operators and their overall probabilities are obtained using
+\texttt{P}. The features can be illustrated on a model of two dice throws:
+\begin{verbatim}
+>>> from sympy.stats import Die, P, E
+>>> X, Y = Die("X"), Die("Y")
+>>> P(Eq(X, 6) & Eq(Y, 6))
+1/36
+>>> P(X>Y)
+5/12
+\end{verbatim}
+The conditions can also be supplied as a second parameter to \texttt{E},
+\texttt{P}, and other methods to calculate the property given the condition:
+\begin{verbatim}
+>>> E(X, X+Y<5)
+5/3
+\end{verbatim}
+
+Using the facilities of the \texttt{sympy.stats} module, one can, for
+example, calculate 
+%Using the above-mentioned features, it is easy to calculate, for example
+the well known properties of maxwellian velocity distribution
+\begin{verbatim}
+>>> from sympy.stats import Maxwell, density
+>>> kT, m, x = symbols("kT m x", positive=True)
+>>> v = Maxwell("v", sqrt(kT/m))
+>>> E(v)                                # mean velocity
+2*sqrt(2)*sqrt(kT)/(sqrt(pi)*sqrt(m))
+>>> E(m*v**2/2)                         # mean energy
+3*kT/2
+>>> solve(density(v)(x).diff(x), x)[0]  # most probable velocity
+sqrt(2)*sqrt(kT)/sqrt(m)
+\end{verbatim}
+
+More information on the \texttt{sympy.stats} module can be found in
+\cite{StatsMRocklin}.
```

Comment:
@rouckas All right, I see your point.  Indeed, the citation style varies in different domains.  AFAIK, in maths and computer science the form "in [x]" is quite widespread and accepted.  On the other hand, the way citations are typeset is given by the bibliography style (a LaTeX style file), so I'd go with the following strategy: if the target journal has a recommended bibliography style, we take; if not, we go with what we have now to avoid the extra effort of re-typesetting all the citations :-)

@asmeurer I'd recommend using "in~\cite{...}" because it prevents line breaks between the "in" and the citation number, which (I think) is considered ugly.